packages/rxjs/spec/observables/from-spec.ts

```diff
@@ -7,7 +7,7 @@ import { ReadableStream } from 'web-streams-polyfill';
 import { observableMatcher } from '../helpers/observableMatcher';
 
 function getArguments<T>(...args: T[]) {
-  return arguments;
```

Comment:
This isn't correct. We actually wanted to test to make sure that `from(arguments)` worked (this is an old pattern that I suspect most people don't use), but it's really testing "ArrayLike", where `arguments` is the most well-used "ArrayLike".